### PR TITLE
Docs ovtf altered

### PR DESCRIPTION
**Pull request by kblaszczak-intel**
the one-page version of OVTF documentation for OV 2022.1

### CODE REVIEW

File: docs/resources/OVTF_overview.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Intel® CPUs
+- Intel® integrated GPUs
+- Intel® Movidius™ Vision Processing Units - referred to as VPU
+- Intel® Vision Accelerator Design with 8 Intel Movidius™ MyriadX VPUs - referred to as VAD-M or HDDL
+
+> **NOTE**: For maximum performance, efficiency, tooling customization, and hardware control, we recommend developers to adopt native OpenVINO™ solutions.
+
+
+To find out more about the product itself, as well as learn how to use it in your project, check its dedicated GitHub repository: 
+(https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/docs)
```
Comment (andrew-zaytsev) [inline docs/resources/OVTF_overview.md:21]:
```suggestion
```

File: docs/resources/OVTF_overview.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+**OpenVINO™ integration with TensorFlow** accelerates inference across many AI models on a variety of Intel® technologies, such as:
+- Intel® CPUs
+- Intel® integrated GPUs
+- Intel® Movidius™ Vision Processing Units - referred to as VPU
+- Intel® Vision Accelerator Design with 8 Intel Movidius™ MyriadX VPUs - referred to as VAD-M or HDDL
+
+> **NOTE**: For maximum performance, efficiency, tooling customization, and hardware control, we recommend developers to adopt native OpenVINO™ solutions.
+
+
+To find out more about the product itself, as well as learn how to use it in your project, check its dedicated GitHub repository:
```
Comment (andrew-zaytsev) [inline docs/resources/OVTF_overview.md:20]:
```suggestion
To find out more about the product itself, as well as learn how to use it in your project, check its dedicated [GitHub repository]((https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/docs). 
```

File: docs/resources/OVTF_overview.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+To find out more about the product itself, as well as learn how to use it in your project, check its dedicated GitHub repository: 
+(https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/docs)
+
+
+To see what you can do with **OpenVINO™ integration with TensorFlow**, explore the demos located in the [examples folder](https://github.com/openvinotoolkit/openvino_tensorflow/tree/master/examples) in our GitHub repository.  
+
+Sample tutorials are also hosted on [Intel<sup>®</sup> DevCloud](https://www.intel.com/content/www/us/en/developer/tools/devcloud/edge/build/ovtfoverview.html). The demo applications are implemented using Jupyter Notebooks. You can interactively execute them on Intel® DevCloud nodes, compare the results of **OpenVINO™ integration with TensorFlow**, native TensorFlow, and OpenVINO™. 
+
+## License
+**OpenVINO™ integration with TensorFlow** is licensed under [Apache License Version 2.0](LICENSE).
```
Comment (andrew-zaytsev) [inline docs/resources/OVTF_overview.md:29]: Shoukd it be a  link to LICENSE?

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

**Comment by kblaszczak-intel** [comment]
Changes moved to a new PR together with structure changes enabling this implementation.

**Pull request closed without merging**